package.json

```diff
@@ -76,7 +76,7 @@
     "dtslint": "^0.9.8",
     "istanbul": "^1.1.0-alpha.1",
     "mocha": "^6.2.1",
-    "mongoose": "^5.4.0",
+    "mongoose": ">=5.7.5",
```

Comment:
What is vulnerable in prior versions?  The changelog doesn't identify any vulnerability-related item.

Also, the peer dependency is at 5.7.3 - is there a reason that isn't updated here?  What is this actually fixing?

Reply:
Those are good questions, a little more context would be helpful. The comment I had is that `>=` isn't the correct SemVer indicator, it should be `^`

Reply:
https://nvd.nist.gov/vuln/detail/CVE-2019-17426